src/modules/users/types/qualification-user-input.js

```diff
@@ -0,0 +1,10 @@
+const { schemaComposer } = require('graphql-compose')
```

Comment:
This is incorrectly implemented. If a qualification relates to a user, a `user` field of type `objectID` should be added to the `qualification` schema.

Reply:
it is input type that is provided to resolver to check whether particular user has particular skill.